spring-boot/starter/src/main/java/org/togglz/spring/boot/actuate/autoconfigure/TogglzAutoConfiguration.java

```diff
@@ -255,7 +255,6 @@ public TogglzDialect togglzDialect() {
     @Configuration
     @ConditionalOnClass({OncePerRequestFilter.class})
     @ConditionalOnMissingBean(HttpServletRequestHolderFilter.class)
-    @ConditionalOnProperty(prefix = "togglz.web", name = "request-holder-filter", havingValue = "true")
```

Comment:
This is acutally the place I would add this:

@ConditionalOnProperty(prefix = "togglz.web", name = "request-holder-filter", havingValue = "true", matchIfMissing = true)

This would make the filter enabled per default, so all config changes made by you in this PR in the samples project are fine, but we would probably need to set the config value to false for the war sample application?